pkg/cmd/server/api/types.go

```diff
@@ -36,6 +38,13 @@ type NodeConfig struct {
 
 	// AllowDisabledDocker if true, the Kubelet will ignore errors from Docker.  This means that a node can start on a machine that doesn't have docker started.
 	AllowDisabledDocker bool
+
+	// PodManifestPath specifies the path for the pod manifest file(s)
+	// The path could point to a single file or a directory that contains multiple manifest files
+	// This is used by the Kubelet to create pods on the node
+	PodManifestPath string
```

Comment:
These should be in their own struct, similar to https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/types.go#L62.  That way they'll always be logically grouped regardless of serialization and if you make it a pointer it becomes clear that it is optional.